### DISCUSSION TITLE
recording inputs

### DISCUSSION THREAD

**Discussion by Dannyg461** [nirgoren/Eddienput#37] [Technical support]
From what I've read it says the program can listen to your inputs from your controller  but pressing f10 on the keyboard doesnt seem to do anything and then pressing select doesn't seem to save any sort of file anywhere, at least not that I can find. Anyone actually know how recording inputs is supposed to work?

**Comment by nirgoren** [comment]
This feature is available on the v2.0 beta release:
https://github.com/nirgoren/Eddienput/releases/tag/v2.0-beta

You are welcome to try it and post your feedback.
